File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/model/playerBattleResult/AllPlayerBattleResultModel.java
package com.syahputrareno975.cardbattlemodule.model.playerBattleResult;

import java.io.Serializable;
import java.util.ArrayList;

public class AllPlayerBattleResultModel implements Serializable {

    public ArrayList<PlayerBattleResultModel> Results = new ArrayList<PlayerBattleResultModel>();

    public AllPlayerBattleResultModel() {
    }

    public AllPlayerBattleResultModel(ArrayList<PlayerBattleResultModel> results) {
        Results = results;
    }
}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/model/playerBattleResult/AllPlayerBattleResultModel.kt
package com.syahputrareno975.simplecardbattle.model.playerBattleResult

import java.io.Serializable

class AllPlayerBattleResultModel : Serializable {

    var Results = ArrayList<PlayerBattleResultModel>()

    constructor()

    constructor(Results: ArrayList<PlayerBattleResultModel>) {
        this.Results = Results
    }
}